internal/requestbody/service.go

```diff
@@ -0,0 +1,30 @@
+package requestbody
+
+import (
+	"encoding/json"
+	"github.com/speakeasy-api/speakeasy-auth-test-service/internal/utils"
+	"io"
+	"net/http"
+
+	"github.com/speakeasy-api/speakeasy-auth-test-service/pkg/models"
+)
+
+func HandleRequestBodyArrOfObj(w http.ResponseWriter, r *http.Request) {
```

Comment:
If this is just going to reflect what it's sent why give it a name that implies a certain type of body? Also I wouldn't even bother deserialising the body and instead pass the bytes straight through to the output unless you need to make sure it's valid json then just deserialise to an interface{}

Reply:
@TristanSpeakEasy I was actually thinking to make the path less generic (ie. `/requestbody/arrOfObj`) so it fits this method name. The purpose would be to only reflect a single field. If I reflect the entire request body, then when I handle in the TS method `httpRes.data == { field1: fieldVal1, field2: fieldVal2, ... , fieldN: fieldValN }`, which means I'll need the response to be an object with the array case being a field in it.

That would be very similar to the array _in_ obj test we're creating in the other PR, and generally how we're treating all other response bodies in our tests so far.

Reply:
Sorry I missed a crucial part, why don't you send an array of objects as well and then it just reflects what it receives nothing is stopping you from sending any shape of data (and if there is something in the sdk then it needs to be fixed)

Reply:
basically if this is a generic endpoint and handles any json input and reflects it back as it receives it, it gives us lots of options for testing with

Reply:
I'm still not understanding or maybe misrepresenting your suggestion. If I reflect back the request body as I receive it, the Typescript SDK will have to handle the entire object, which means `class-transformer` kicks in, so in this scenario we'd be testing the class<arr<class>> case instead of arr<class>. That is my reasoning for deserializing and picking a specific field of the body.

Are you suggesting we reflect the request body, but deserialize and pass the field through to send both? Sorry for the confusion

Reply:
So if you define your request body like:

```yaml
      requestBody:
        content:
          application/json:
            schema:
              type: array
              items:
                type: object
                properties:
                  name:
                    type: string
                  age:
                    type: integer
```

Then the server will receive a json array:

```
[{"name": "tristan", "age": "too old"}]
```

as the body which you can send straight back as the response

Reply:
cool I see your suggestion -- fixed and tested